tests/dialects/test_bigquery.py

```diff
@@ -603,11 +614,11 @@ def test_bigquery(self):
             read={"spark": "select posexplode_outer([])"},
         )
         self.validate_all(
-            "SELECT AS STRUCT ARRAY(SELECT AS STRUCT b FROM x) AS y FROM z",
+            "SELECT AS STRUCT ARRAY(SELECT AS STRUCT 1 AS b FROM x) AS y FROM z",
```

Comment:
Did this need to change? I think both this and the previous one are valid BQ queries, but I'm curious if the old one can't be transpiled for some reason?

Reply:
the issue is if you do it it's an unnamed struct so i changed it